Sources/LocalizedStringKit/include/LocalizedStringKit.h

```diff
@@ -14,25 +14,34 @@ FOUNDATION_EXPORT const unsigned char LocalizedStringKitVersionString[];
 
 /// Primary localization function used to localize strings
 ///
-/// The `value` should be the English string and the `comment` should give context on where the string is used.
-/// Ex: `Localized("Cancel", "Action sheet action title")
-NSString *Localized(NSString *value, NSString *comment);
+/// @param value: The English string
+/// @param comment: String to give context where the value string is used
+/// @param tableName: Optional string to provide additional classification of the value string. Can be used to segment groups of strings in multiple bundles.
+///
+/// Examples
+/// Localized("Cancel", "Action sheet action title", nil)
+/// Localized("Cancel", "Action sheet action title", "primary")
+/// Localized("Cancel", "Action sheet action title", "primary")
+NSString *Localized(NSString *_Nonnull value, NSString *_Nonnull comment, NSString *_Nullable tableName);
```

Comment:
Can you add an additional convenience method for users who want to avoid the table name functionality, that just sets `tableName` to nil?

```objc
NSString *Localized(NSString *_Nonnull value, NSString *_Nonnull comment);
```

Reply:
I don't know if you can do that (easily) in a C-based language?

Reply:
Yep, sure thing.

Reply:
Ah yes. The names conflict. I'll go ahead and update the API with a tableName to have a new name.

Reply:
@a2 it would add a little more duplication but it should be as simple as

```swift
NSString *Localized(NSString *_Nonnull value, NSString *_Nonnull comment) {
  return [LocalizedStringKit localizeWithValue:value comment:comment keyExtension:nil tableName:nil];
}
```

Reply:
Oh I see what you mean now :), that makes sense